### PR TITLE
Merge swift-5.1 into stable

### PR DESCRIPTION
**Pull request by dcci**
Stefan, this was really behind, so I took the freedom to merge.

### CODE REVIEW

**Comment by dcci** [comment]
@swift-ci test and merge

File: source/Symbol/SwiftASTContext.cpp [hunk 1]
```diff
@@ -1276,74 +1284,93 @@ static void ConfigureResourceDirs(swift::CompilerInvocation &invocation,
...
-    break;
-
+    return "The module file format is too new to be used by this version of "
+           "the debugger.";
   case swift::serialization::Status::MissingDependency:
-    error << "the swift module file depends on another module that can't be "
-             "loaded\n";
-    break;
-
-  case swift::serialization::Status::MissingUnderlyingModule:
```
Comment (weliveindetail) [inline source/Symbol/SwiftASTContext.cpp]: This is the new name for `MissingShadowedModule` https://github.com/apple/swift/commit/6cca050fd9

**Review by weliveindetail — COMMENTED** [review]
Otherwise LGTM. Thanks

**Comment by weliveindetail** [comment]
@swift-ci test and merge

**Pull request merged**